burgerBlockchain.js

```diff
@@ -56,7 +55,7 @@ class BurgerBlockchain {
     }
 
     addBlock(block) {
-        this.cumulativeDifficulty += block.difficulty; // temporary implementation while waiting for Issue #46
+        this.cumulativeDifficulty += this._getCumulativeDifficultyValue(block.difficulty);
```

Comment:
in the case of adding the block, I think it is quicker to just add the cumulative difficulty of the submitted block instead of calculating the cumulative difficulty of the whole chain.

Reply:
why do I have a feeling that that could backfire on us?

Reply:
hmmm... I haven't tested this scenario yet.

Should we change this to calculating the whole chain instead of just adding the cumulative difficulty?

Reply:
well, i'm saying we should provide that information on a need to know basis. I don't see a real need to keep track of the cumulative difficulty. But you can leave it as is for now really...